packages/gatsby/src/redux/actions.js

```diff
@@ -347,12 +345,6 @@ actions.deleteNodes = (nodes: any[], plugin: Plugin) => {
     plugin,
     payload: nodes,
   }
-
-  if (deleteDescendantsActions) {
```

Comment:
Removed this as eslint marked it as unreachable. Should deleting descendants still be included?

Reply:
Yeah — this might be a bad merge. If you look at `deleteNode`, it finds descendant nodes to delete above.

Reply:
I've reverted this, handling it in #5106 instead.